packages/react/src/progress-bar/ProgressBar.tsx

```diff
@@ -2,8 +2,12 @@ import * as ProgressPrimitive from "@radix-ui/react-progress";
 import { type ComponentPropsWithRef, type ElementRef, forwardRef } from "react";
 
 import { Box } from "../box";
+import { type ExtendProps } from "../utils";
 
-type ProgressBarProps = ComponentPropsWithRef<typeof ProgressPrimitive.Root>;
+type ProgressBarProps = ExtendProps<
+  ComponentPropsWithRef<typeof ProgressPrimitive.Root>,
+  ComponentPropsWithRef<typeof Box>
+>;
 
 export const ProgressBar = forwardRef<
```

Comment:
sorry but can we rename this component to just `Progress` ?

that way it'll be consistent with the radix primitive - also that's what it's called in majority of other component libraries